client/css/splitpanel.css

```diff
@@ -0,0 +1,33 @@
+
+.silky-splitpanel-splitter {
+    cursor: ew-resize;
+}
+
+
+.silky-sp-back-button {
+    font-size: 24px ;
+    width: 40px ;
+    height: 40px ;
+    //float: right;
```

Comment:
`//` isn't actually a comment in css, only `/*  */` are comments (i'm pretty sure).